app/src/main/java/net/squanchy/navigation/BottomNavigationHelper.java

```diff
@@ -0,0 +1,31 @@
+package net.squanchy.navigation;
+
+import android.support.design.internal.BottomNavigationItemView;
+import android.support.design.internal.BottomNavigationMenuView;
+import android.support.design.widget.BottomNavigationView;
+import android.util.Log;
+
+import java.lang.reflect.Field;
+
+class BottomNavigationHelper {
+
+    //Hack
+    static void disableShiftMode(BottomNavigationView view) {
```

Comment:
Or we open a PR 😁 I would add it to the list of things we can make better in there...